evergreen.yml

```diff
@@ -43,10 +43,27 @@ functions:
             export CARGO_NET_GIT_FETCH_WITH_CLI="$CARGO_NET_GIT_FETCH_WITH_CLI"
             export UNIX_ODBC_PATH="$PWD/unixodbc/install"
             git config --global url."ssh://git@github.com/".insteadOf "https://github.com/"
+            export ADL_TEST_USER=test_user
+            export ADL_TEST_PWD=test_pwd
+            export ADL_TEST_HOST=test_host
+            export ADL_TEST_DB=test_db
           EOT
     - command: expansions.update
       params:
-        file: mongo-odbc-driver/expansions.yml
+        file: mongosql-odbc-driver/expansions.yml
+
+  "add Windows DLL path":
+    - command: shell.exec
+      params:
+        shell: cmd
+        working_dir: mongosql-odbc-driver
+        script: |
+          set WIN_DRIVER_DLL_PATH=%cd%
+          set WIN_DRIVER_DLL_PATH=%WIN_DRIVER_DLL_PATH:\=\\\\\\\\%
+          echo   export DRIVER_DLL_PATH="%WIN_DRIVER_DLL_PATH%" >> expansions.yml
+    - command: expansions.update
+      params:
+        file: mongosql-odbc-driver/expansions.yml
```

Comment:
This seems like a very brittle way of getting the `DRIVER_DLL_PATH` export into the `prepare_shell` expansion, since it assumes that `expansions.yml` is formatted in a specific way. Is there any reason we can't just do this in the existing `generate expansions` function above?

Reply:
I was doing it this way because I wanted to get the Windows path and also thought that I needed DRIVER_DLL_PATH between tasks. I found the cygpath command and also realized that DRIVER_DLL_PATH is only used in the driver setup task so it doesn't have to be here.